fix: Trigger frontend build for LSE

makseq opened the pull request:
<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

makseq committed:
fix: Trigger frontend build for LSE

makseq committed:
Merge branch 'develop' of github.com:heartexlabs/label-studio into fb-trigger-frontend-build

farioas commented:
/fm merge
> Command | Description
> --- | ---
> /fm sync <Optional branch override> | Sync upstream prs and merge with pull request base

farioas commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/15541365273)

makseq committed:
Linter

farioas committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15541365273

hlomzik commented on poetry.lock:
Is this intentional change?

robot-ci-heartex commented:
This PR is stale because it has been open 45 days with no activity. Remove `stale` label or comment or this will be closed in 10 days.

robot-ci-heartex commented:
This PR was closed because it has been stalled for 10 days with no activity.
